src/app/api/cms/messages/get_message.ts

```diff
@@ -1,68 +1,56 @@
 import ActiveITSClient from "@entities/ActiveITSClient";
+import ActiveITSDBClient from "@entities/ActiveITSDBClient";
 import * as env from "@utils/config.js";
-import fs from "fs";
-import { Builder, parseString } from "xml2js";
+import { NextResponse, type NextRequest } from "next/server";
 
 const api_url: string | undefined = process.env.ACTIVEITS_API_HOSTNAME;
 const api_port: string | undefined = process.env.ACTIVEITS_API_PORT;
 
-export default async function get_message(req: Request, res: Response) {
+export default async function get_message(req: NextRequest, res: NextResponse) {
+  const dict_response: Record<string, string> = {};
   try {
-    const active_client: ActiveITSClient = new ActiveITSClient(
-      api_url!,
-      Number(api_port)
-    );
-
-    // Connect to the client
+    const active_client: ActiveITSClient = new ActiveITSClient(api_url!, Number(api_port));
+    const db_client: ActiveITSDBClient = ActiveITSDBClient.getInstance();
```

Comment:
Very minor naming nitpick, but since `ActiveITSDBClient` is a sort of factory with static methods now, maybe the class name should instead be `ActiveITSDB` and `getInstance` should be renamed `getClient`.

e.g. `const dbClient = ActiveITSDB.getClient()` instead of `const db_client: ActiveITSDBClient = ActiveITSDBClient.getInstance();`